Documentations/set_manufacturing_data.py

```
@@ -0,0 +1,243 @@
+        # Set Bluetooth name based on serial
+        set_bluetooth_name(ser)
+        
+        print("\nAll done! The device will now configure the Bluetooth module.")
+        print("Wait for the bootloader to finish and reconnect.")
+        
+        ser.close()
+        
+    except serial.SerialException as e:
+        print(f"Serial error: {e}")
+        sys.exit(1)
+    except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)